home/static/home/CSS/base_style.css

```diff
@@ -40,3 +40,14 @@ body {
 	padding-left:5px;
 	padding-right:5px;
 }
+
+
+a.nostyle:link {
+    text-decoration: inherit;
+    color: inherit;
+}
+
+a.nostyle:visited {
+    text-decoration: inherit;
+    color: inherit;
+}
```

Comment:
The style was added to base_style.css because I want to use this style in different pages, so placing it here will allow me to use it from every where